mysql/README.md

```diff
@@ -0,0 +1,233 @@
+| `CHAR(N)`                                         | `STRING(N)`     | c                               |
+| `DATE`                                            | `DATE`          |                                 |
+| `DATETIME`                                        | `TIMESTAMP`     | t                               |
+| `DECIMAL`, `NUMERIC`                              | `FLOAT64`       | p                               |
+| `DOUBLE`                                          | `FLOAT64`       |                                 |
+| `ENUM`                                            | `STRING(MAX)`   |                                 |
+| `FLOAT`                                           | `FLOAT64`       | s                               |
+| `INTEGER`, `MEDIUMINT`,<br/>`TINYINT`, `SMALLINT` | `INT64`         | s                               |
+| `JSON`                                            | `STRING(MAX)`   |                                 |
+| `SET`                                             | `ARRAY<STRING>` | SET only supports string values |
+| `TEXT`, `MEDIUMTEXT`,<br/>`TINYTEXT`, `LONGTEXT`  | `STRING(MAX)`   |                                 |
+| `TIMESTAMP`                                       | `TIMESTAMP`     |                                 |
+| `VARCHAR`                                         | `STRING(MAX)`   |                                 |
+| `VARCHAR(N)`                                      | `STRING(N)`     | c                               |
+
+Spanner does not support `spatial` datatypes of MySQL. Along with `spatial` datatypes,
+all other types map to `STRING(MAX)`. Some of the mappings in this table
+represent loss of precision (marked p), differences in treatment of timezones (marked t), differences in
+treatment of fixed-length character types (marked c), and changes in storage
+size (marked s). We discuss these, as well as other limits and notes on
+schema conversion, in the following sections.
+
+### `DECIMAL` and `NUMERIC`
+
+Spanner does not support decimal types, so these are mapped to `FLOAT64`. For
+some decimal types (e.g. `DECIMAL(7, 3))` this mapping will preserve precision.
+But for others, the decimal type does not fit in `FLOAT64`; HarbourBridge
+generates a warning in such cases. In general, mapping `DECIMAL` to `FLOAT64`
+can be useful for evaluation purposes, but it is not recommended for production
+use. Note that in MySQL, NUMERIC is implemented as DECIMAL, so the remarks about DECIMAL apply equally to NUMERIC.
+
+### `TIMESTAMP` and `DATETIME`
+
+MySQL has two timestamp types: `TIMESTAMP` and `DATETIME`. Both provide microsecond resolultion, but neither
+actually stores a timezone with the data. The key difference between the two types is that MySQL converts `TIMESTAMP` values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. This does not occur `DATETIME` and data is returned
```

Comment:
Can we reformat this line and other lines in this file to max line length of 80 characters?